pkglistgen/update_repo_handler.py

```diff
@@ -199,6 +199,8 @@ def update_project(apiurl, project):
         key = str(key)
         if not opts['url'].endswith('/'):
             opts['url'] += '/'
+        if 'downloadcontent.opensuse.org' in opts['url']:
+            opts['url'].replace('downloadcontent.opensuse.org', 'download.opensuse.org')
```

Comment:
We intentionally picked downloadcontent.o.o to not being redirected wildly across the internet to grab downloads for our update stacks. we know we have the stuff on our own mirror